src/main/proto/service.proto

```diff
@@ -69,3 +71,14 @@ message CreateCustomerRoleRequest {
 message CreateCustomerRoleResponse {
   optional bytes role_id = 1;
 }
+
+message MoveAccountBalanceRequest {
+  optional bytes from_account_id = 1;
+  optional bytes to_account_id = 2;
```

Comment:
This applies to existing methods as well, so it's likely a tangential discussion: Bytes are a bit unwieldy when you are trying to locally test stuff out and adds some complexity in client side code. Should we consider changing these to int64. The storage format can still be Bytes.

@voulg What do you think ?

Reply:
That would be nice, unfortunately UUID is 128 bit :(

I agree with this: https://stackoverflow.com/questions/36344826/how-do-i-represent-a-uuid-in-a-protobuf-message We should either use bytes, or strings. Strings are more readable and there's a standard to map UUIDs to strings, but longer.

We have a similar suggestion for how to encode UUIDs in Spanner and I went with BYTES for storage efficiency, but it's likely for the gRPC using string might be better. Thoughts?

Reply:
Ah I see. Yeah, I kind of like the human readable string for the RPC.

Reply:
sgtm, Nick, Bianca, thoughts?

In any case changing to strings would require a new PR focused just in that, perhaps we can file an issue.

Reply:
right now i think we've been using bytes for any UUIDs. im definitely ok with changing it, but if we do, we should move this to a separate PR because this means changing a few methods in `FinAppService` that would crowd this PR.

Reply:
Yeah, I'm all here for switching to strings in a separate PR. See https://github.com/GoogleCloudPlatform/cloud-spanner-samples/issues/32